README.md

```diff
@@ -1,197 +1,212 @@
+   + You can manually push the DynamoDB data to Elasticsearch by creating a lambda instance.
+   + You can refresh your DynamoDB table with a backup.
+   + You can remove all data from the DynamoDB table and that will create parity between Elasticsearch and DynamoDB.
 
-  1. You can manually redrive the DynamoDB data to Elasticsearch by creating a lambda
-  1. You can refresh your DynamoDB table with a back-up
-  1. You can remove all data from the DynamoDB table and that will create parity between Elasticsearch and DynamoDB
++ Support for STU3 and [R4](https://www.hl7.org/fhir/validation.html) releases of FHIR is based on the JSON schema provided by HL7. The schema for R4 is more restrictive than the schema for [STU3](http://hl7.org/fhir/STU3/validation.html). The STU3 schema doesn’t restrict appending additional fields into the POST/PUT requests of a resource, whereas the R4 schema has a strict definition of what is permitted in the request. You can access the schema [here](https://github.com/awslabs/fhir-works-on-aws-routing/blob/mainline/src/router/validation/schemas/fhir.schema.v3.json).
 
-- Support for STU3 and R4 releases of FHIR is based on the JSON schema provided by HL7. The schema for [R4](https://www.hl7.org/fhir/validation.html) is more restrictive than the schema for [STU3](http://hl7.org/fhir/STU3/validation.html). The STU3 schema doesn’t restrict appending additional fields into the POST/PUT requests of a resource, whereas the R4 schema has a strict definition of what is permitted in the request.
+**Note**: We are using the official schema provided by [HL7](https://www.hl7.org/fhir/STU3/downloads.html). 
 
-- When making a POST/PUT request to the server, if you get an error that includes the text `Failed to parse request body as JSON resource`, check that you've set the request headers correctly. The header for `Content-Type` should be either `application/json` or `application/fhir+json` If you're using Postman for making requests, in the `Body` tab, be sure to also set the setting to `raw` and `JSON`.
-  ![Postman Body Request Settings](resources/postman_body_request_settings.png)
++ When making a `POST`/`PUT` request to the server, if you get an error that includes the text `Failed to parse request body as JSON resource`, check that you've set the request headers correctly. The header for `Content-Type` should be either `application/json` or `application/fhir+json` If you're using Postman for making requests, in the Body tab, be sure to also set the setting to `raw` and `JSON`.
+![Postman Body Request Settings](resources/postman_body_request_settings.png)
 
 ## Feedback
-
-We'd love to hear from you! Please reach out to our team: [fhir-works-on-aws-dev](mailto:fhir-works-on-aws-dev@amazon.com) for any feedback.
+We'd love to hear from you! Please reach out to our team: [fhir-works-on-aws-dev](mailto:fhir-works-on-aws-dev@amazon.com) for any feedback.
```

Comment:
I believe in our retro we decided to remove our email. Let's rephrase this to have them file GH issues if they have any questions or concerns.

Reply:
fixed